docs/examples/ButtonBlock.jsx

```diff
@@ -0,0 +1,8 @@
+var buttonInstance = (
```

Comment:
I wonder if it's worth changing all these examples to something standard like `var example = (` instead. Otherwise we have to think about naming things and keeping them up to date, see Button.jsx, that's no longer a `buttonInstance`, that's a collection of `buttonInstance`s.